### ISSUE TITLE
[Bug]: ut Test_NewServer_TLS failed

### ISSUE THREAD

**Issue by longjiquan** [milvus-io/milvus#17201]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

https://github.com/milvus-io/milvus/runs/6569041998?check_suite_focus=true

**Comment by yanliang567** [comment]
@longjiquan i think you are working on fix 

/assign @longjiquan 
/unassign

**Comment by longjiquan** [comment]
Fixed by #17208
